5.2 - Clarified CA certificate migration requirements (#5133)
uyuni-project/uyuni-docs: en/modules/installation-and-upgrade/partials/snippet-certificate-validators.adoc

Strict certificate validators, including Python 3.13 and later, reject CA certificates that miss the `critical` flag.

If your installation uses an older self-signed CA generated by {productname}, you might need to create a new CA and new server certificates, then deploy the new root CA to all clients.

> You can rotate the CA certificate on a Podman installation using the new `mgradm ssl` command. For more information, see xref:administration:ssl-certs-ca-migration.adoc[].

Reviewer: ```suggestion
You can rotate the CA certificate on a Podman installation using the new `mgradm ssl` command. 
For more information, see xref:administration:ssl-certs-ca-migration.adoc[].
```

> Note that the command can only be used *after* upgrading both `mgradm` and {productname}. It is perfectly fine to upgrade both even if affected, as long as the CA certificates are rotated and recognized by clients before the server is started.

Reviewer: ```suggestion
Note that the command can only be used *after* upgrading both `mgradm` and {productname}. 
You can safely upgrade affected installations as long as you rotate the CA certificates and ensure clients recognize them before starting the server.
```

> Not rotating the CA certificate may not be fatal, but is heavily discouraged.

Reviewer: ``` suggestion
While failing to rotate the CA certificate may not cause an immediate system failure, it is strongly discouraged.
```
